internal/examples/semantics/generated_test.go

```diff
@@ -194,7 +194,7 @@ func (suite *GoTestSuite) TestBreakFromLoopNoContinue() {
 func (suite *GoTestSuite) TestBreakFromLoopNoContinueDouble() {
 	d := disk.NewMemDisk(30)
 	disk.Init(d)
-	suite.Equal(true, failing_testBreakFromLoopNoContinueDouble())
+	suite.Equal(true, testBreakFromLoopNoContinueDouble())
```

Comment:
I am not quite sure what this file is about... it says "generated" but I hand-edited this change since CI complained about a diff here.

Are these Coq-Go-conformance tests run automatically on CI?

Reply:
The comment at the top of the file is quite clear about not editing this by hand. But then how to invoke test_gen? THe README says "To re-generate the Go test file you can just run go generate ./...". So I did `go generate internal/examples/semantics/generated_test.go` but that does not seem to do anything...

Reply:
This seems to work:
```
go run ./cmd/test_gen -go -out ./internal/examples/semantics/generated_test.go ./internal/examples/semantics
```